gdcdictionary/schemas/_definitions.yaml

```diff
@@ -13,13 +13,43 @@ parent_uuids:
     $ref: "#/uuid"
   uniqueItems: true
 
+foreign_key_project:
```

Comment:
Just curious if you know how `planX` uses these properties, given `to_one` and `to_many` is generic enough to cover same use cases and these additions only redefine existing properties with a name change.

Reply:
They use "code" instead of "submitter_id" in foreign_key. That's the only difference. Please look up any planX dictionary.